geo-types/src/line_string.rs

```diff
@@ -1,10 +1,9 @@
-#[cfg(any(feature = "approx", test))]
-use approx::{AbsDiffEq, RelativeEq};
-
-use crate::{CoordNum, Coordinate, Line, Point, Triangle};
+use crate::*;
 use std::iter::FromIterator;
 use std::ops::{Index, IndexMut};
 
+/// A generic line string with 3D space + Measure value support.
```

Comment:
Can you rephrase this (and all the similar geometry docs that you're updating) so that they remain geared primarily towards 2d people, with an addendum about 3D/M support.

Something like:
```
/// An ordered collection of two or more [`Coordinate`]s, representing a
/// path between locations.
///
/// LineStrings are 2D by default, but optionally support 3D and Measure parameters.
```

The overwhelming majority of people are going to be doing 2D, so we should keep are docs optimized for them.

Reply:
fixed